python/tchannel/handler.py

```diff
@@ -97,16 +98,23 @@ def handle_pre_call(self, message_id, message, connection):
         """
         try:
             req = connection.request_message_factory.build(message_id, message)
-        except InvalidChecksumException as e:
+            # call handler only for the call request message
+            # not continue message
+            if req:
```

Comment:
it will be none, when we receive callreqcontinue message. We only build request when we receive callrequest message,